packages/GA4Client/src/initialization.js

```diff
@@ -46,6 +47,10 @@ var initialization = {
         }
         gtag('config', measurementId, configSettings);
 
+        gtag('get', measurementId, 'client_id', function (clientId) {
+            setClientId(clientId, this.moduleId);
```

Comment:
It should be `initialization.moduleId` based on where you currently have `moduleId` defined.  This will otherwise throw an error because of the `this` context bc of function syntax vs arrow syntax